packages/site/src/components/App.js

```diff
@@ -60,27 +61,40 @@ const Root = createStyledComponent('div', styles.app);
 const Nav = createStyledComponent(_Nav, styles.nav);
 const Main = createStyledComponent('main', styles.main);
 
+const getFirstDemo = (demos: Object): string => {
+  for (const first in demos) {
+    return first.toString();
+  }
+  return '';
+};
+
 export default function App({ className, demos }: Props) {
-  if (demos.slug) {
+  let siteDemos = {};
```

Comment:
Flow yells at you because `demos` in that first line can be an object or an array. This works, though:

```
if (!Array.isArray(demos) && demos.slug) {
  return <ComponentDoc {...demos} />;
}

const siteDemos = Array.isArray(demos) ? pullOutKeys(demos, 'slug') : demos;
```

Still like that better than what's currently there?